### ISSUE TITLE
Unexpected behaviour when using mpl_style with plt.savefig()

### ISSUE THREAD

**Issue by PsiPhiTheta** [quantumblacklabs/qbstyles#8]
Consider the following minimal working example which demonstrates unexpected behaviour when using `qbstyles.mpl_style` with `plt.savefig()`:

```
import matplotlib.pyplot as plt
from qbstyles import mpl_style
mpl_style(dark=True)
a = [1,2,3,4,5]
plt.plot(a)
plt.savefig('filename.png')
```

The output image is missing most elements and does not have the qbstyle applied:
![filename](https://user-images.githubusercontent.com/25353324/62858916-9d153b80-bcf3-11e9-9b62-908a89f0cb8a.png)

**Comment by tsanikgr** [comment]
Hello @PsiPhiTheta.

Are you trying to plot in a Jupyter notebook?
Have you read [this](https://github.com/quantumblacklabs/qbstyles#how-do-i-use-qb-styles-in-jupyter-notebooks) warning on our README?

I cannot reproduce your issue otherwise, any further info would be much appreciated. Thanks!

**Comment by PsiPhiTheta** [comment]
Hi @tsanikgr,

Thanks for your help. Actually I am not using Jupyter, I am using Python v3.7, matplotlib v3.1.1, qbstyles v0.1.2 via the PyCharm IDE v2019.2.1 for macOS.

Happy to provide any further info as needed.

**Comment by tsanikgr** [comment]
Hello @PsiPhiTheta,

Thanks for spotting this bug! `plt.show()` works, but as you correctly identified `plt.savefig()` was not applying the `facecolor`. In the `dark` theme, this defaulted to white so you could not see anything.

I just merged #9 which fixes this bug (and another one).

We will release to `pypi` in a couple of hours, after which you 'll be able to update using `pip install qbstyles -U`.

In the meantime, if you would like to try the fix you can clone the repo and do `pip install . -U`

Thanks again!

**Comment by PsiPhiTheta** [comment]
Thanks @tsanikgr, I'll test this fix tomorrow and will report the results back here.

**Comment by tsanikgr** [comment]
We released and it works on my machine. Just update with `pip install qbstyles -U`

Thanks again for reporting :)

---
Fixed by #9

**Comment by PsiPhiTheta** [comment]
Also working on my end, cheers @tsanikgr.
